pymerger/merger.py

```diff
@@ -261,6 +263,9 @@ def merge(raw_filepaths, output=None):
     # Parse files
     parsed_files = []
     for file_path in file_paths:
+        if root_path:
```

Comment:
this was amended to the proper place. In the rush, it was placed inside the wrong loop. Please see: https://github.com/gentunian/pymerger/blob/master/pymerger/merger.py#L279